### PR TITLE
Email routes

### PR DESCRIPTION
**Pull request by mark-marshall**
# Description

1) Adds new get chats by guest ID route in chatsRoutes
2) Adds new sendEmail route in emailRoutes (new route)

Note: Needs to be documented and formally tested

Fixes #73

## Change status

- [ ] Complete, tested, ready to review and merge

- [x] Complete, but not tested (may need new tests)

- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Via Postman, Gmail, and local version of front-end.

Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:

- [x] My code follows the style guidelines of this project

- [x] I have performed a self-review of my own code

- [ ] I have commented my code, particularly in hard-to-understand areas

- [ ] I have made corresponding changes to the documentation, if relevant

- [x] I have run the app using my feature and ensured that no functionality is broken

- [x] My changes generate no new warnings

- [ ] There are no merge conflicts

### CODE REVIEW

File: routes/emailRoutes.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        const msg = {
+          to: guestEmail,
+          from,
+          subject: `${subject} ${hotel.name}`,
+          html: emailBody,
+        };
+        await sgMail.send(msg);
+        res.status(200).json(response.sendChatLog);
+      } else {
+        res.status(200).json(response.noChats);
```
Comment (Pav0l) [inline routes/emailRoutes.js]: `200` 😱
I'd just probably send an empty array when there are no chats and let FE handle it with 
`if (chats.length === 0) return (<div>There are no chats for this guest</div>);`

File: utils/errorMessage.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const errorMessage = {
   invalidUserName: { message: 'Invalid user name' },
   deleteUser: { message: 'The user could not be deleted' },
   addUser: { message: 'The user could not be added.' },
-  superAdminError: { message: 'Can\'t create Super Admin in this endpoint.' },
+  superAdminError: { message: "Can't create Super Admin in this endpoint." },
```
Comment (Pav0l) [inline utils/errorMessage.js]: Was this autoformated?

File: utils/getChatOnCheckout.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const axios = require('axios');
+const path = require('../utils/path');
+const url = require('../utils/url');
+
+const getChatOnCheckout = async guestId => {
+  try {
+    const chat = await axios.get(
+      `http://localhost:7000${path.chats}/checkout/${guestId}`,
+    );
+    // const chat = await axios.get(`${url}${path.chats}/checkout/${guestId}`);
```
Comment (Pav0l) [inline utils/getChatOnCheckout.js]: I think you forgot to un-comment this line and remove the `localhost` one

File: utils/getGuest.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+const axios = require('axios');
+const path = require('./path');
+const url = require('./url');
+
+const getGuest = async guestId => {
+  try {
+    const guest = await axios.get(
+      `http://localhost:7000${path.users}/${guestId}`,
+    );
+    // const guest = await axios.get(`${url}${path.users}/${guestId}`);
```
Comment (Pav0l) [inline utils/getGuest.js]: Same as above

File: utils/getHotel.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+const axios = require('axios');
+const path = require('./path');
+const url = require('./url');
+
+const getGuest = async hotelId => {
+  try {
+    const hotel = await axios.get(
+      `http://localhost:7000${path.hotel}/${hotelId}`,
+    );
+    // const guest = await axios.get(`${url}${path.hotel}/${hotelId}`);
```
Comment (Pav0l) [inline utils/getHotel.js]: 3 x ☝️

**Review by Pav0l — CHANGES_REQUESTED** [review]

File: routes/emailRoutes.js (hunk 1 above)
Comment (mark-marshall) [reply routes/emailRoutes.js]: Much better idea

**Review by Pav0l — APPROVED** [review]

**Pull request merged**